### PR TITLE
Disable InvokeCodeThatShouldFirEvents_EnsureEventsFired

### PR DESCRIPTION
**Pull request by hoyosjs**
Tracked by https://github.com/dotnet/runtime/issues/52710

### CODE REVIEW

File: src/libraries/System.Data.Common/tests/System/Data/DataCommonEventSourceTest.cs [hunk 1]
```diff
@@ -11,6 +11,7 @@ namespace System.Data.Tests
     public class DataCommonEventSourceTest
     {
         [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/52710", TestPlatforms.OSX, TestRuntimes.CoreCLR)]
```
Comment (hoyosjs) [inline src/libraries/System.Data.Common/tests/System/Data/DataCommonEventSourceTest.cs:14]:
```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/52710", TestPlatforms.OSX, runtimes: TestRuntimes.CoreCLR)]
```

**Review by jkoritzinsky — APPROVED** [review]

**Comment by MihaZupan** [comment]
https://github.com/dotnet/runtime/pull/53043 is the fix for the issue, I don't think it's worth disabling the test before that is merged

**Comment by hoyosjs** [comment]
Happy to close this one in favor of that one. I'll review that one.

**Comment by MihaZupan** [comment]
Thanks!

**Pull request closed without merging**